e2e/genpolicy/genpolicy_test.go

```diff
@@ -19,21 +19,25 @@ import (
 	"github.com/edgelesssys/contrast/e2e/internal/contrasttest"
 	"github.com/edgelesssys/contrast/e2e/internal/kubeclient"
 	"github.com/edgelesssys/contrast/internal/kuberesource"
+	"github.com/edgelesssys/contrast/node-installer/platforms"
 	"github.com/stretchr/testify/require"
 )
 
 var (
-	imageReplacementsFile, namespaceFile string
-	skipUndeploy                         bool
+	imageReplacementsFile, namespaceFile, platform string
+	skipUndeploy                                   bool
 )
 
 // TestGenpolicy runs regression tests for generated policies.
 func TestGenpolicy(t *testing.T) {
 	testCases := kuberesource.GenpolicyRegressionTests()
 
+	p, err := platforms.FromString(platform)
```

Comment:
```suggestion
	platform, err := platforms.FromString(platformStr)
```
`p` is to short. One letter vars only for method receivers, loops vars, common things etc.